ui/src/types.ts

```diff
@@ -30,7 +30,7 @@ export type CertificateAuthorityEntry = {
 export type User = {
   exp: number;
   id: number;
-  permissions: number;
+  role_id: number;
```

Comment:
This is very nicely typed in the Go code. Should we do something similar and avoid having magic values like `auth.user?.role_id == 0`?

Reply:
Agreed, I just found how to do that! Done.